htdocs/accountancy/journal/bankjournal.php

```diff
@@ -158,6 +158,7 @@
 if ($in_bookkeeping == 'notyet') {
 	$sql .= " AND (b.rowid NOT IN (SELECT fk_doc FROM ".MAIN_DB_PREFIX."accounting_bookkeeping as ab  WHERE ab.doc_type='bank') )";
 }
+$sql .= " GROUP BY b.rowid";
```

Comment:
If you add a group by, it means you want to make an aggregation function.
But there is non aggregation function in sql. So request will fails on postgres and on mysql with option strict mode on.